commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -66,13 +66,13 @@
 	CLINICAL_DOCUMENT_CONTAINS_DUPLICATE_ECQM_SECTIONS(28, "Clinical Document contains duplicate Measure "
 			+ "sections"),
 	REPORTING_PARAMETERS_MUST_CONTAIN_SINGLE_PERFORMANCE_START(29, "Must have one and only one performance period "
-			+ "start. Please see the Implementation Guide for information on the performance period here: "
+			+ "start. Please see the Implementation Guide for information on the performance period here: " //NOSONAR
 			+ DocumentationReference.PERFORMANCE_PERIOD),
 	REPORTING_PARAMETERS_MUST_CONTAIN_SINGLE_PERFORMANCE_END(30, "Must have one and only one performance period end. "
-			+ "Please see the Implementation Guide for information on the performance period here: "
+			+ "Please see the Implementation Guide for information on the performance period here: " //NOSONAR
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Is //NOSONAR used to exclude false-positive or to hide real quality flaw ? [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ANoSonar)

Reply:
FALSE POSITIVE